front/app/translations/admin/nl-BE.json

```diff
@@ -1307,11 +1308,18 @@
   "app.containers.IdeasShow.proposedBudgetTitle": "Budget",
   "app.containers.IdeasShow.xDaysLeft": "Nog {x, plural, =0 {minder dan een dag} one {één dag} other {# dagen}}",
   "app.containers.admin.ContentBuilder.delete": "Verwijder",
+  "app.containers.admin.ContentBuilder.image": "Afbeelding",
+  "app.containers.admin.ContentBuilder.imageAltTextLabel": "Korte beschrijving van de afbeelding",
+  "app.containers.admin.ContentBuilder.imageAltTextTooltip": "Adding \"alt text\" for images is important to make your platform accessible for users using screen readers.",
   "app.containers.admin.ContentBuilder.oneColumnLayout": "1 kolom",
   "app.containers.admin.ContentBuilder.projectDescription": "Projectbeschrijving",
   "app.containers.admin.ContentBuilder.text": "Tekst",
   "app.containers.admin.ContentBuilder.textValue": "Dit is tekst. Je kunt het aanpassen en formatteren met behulp van de editor in het paneel aan de rechterzijde.",
+  "app.containers.admin.ContentBuilder.threeColumnLayout": "3 column",
   "app.containers.admin.ContentBuilder.twoColumnLayout": "2 kolom",
+  "app.containers.admin.ContentBuilder.twoColumnLayoutVariant1-2": "2 columns with 30% and 60% width respectively",
+  "app.containers.admin.ContentBuilder.twoColumnLayoutVariant2-1": "2 columns with 60% and 30% width respectively",
+  "app.containers.admin.ContentBuilder.twoEvenColumnLayout": "2 even columns",
```

Comment:
@IvaKop I remember you showcased this, so checking with you: is this feature already visible? Translations missing for some languages.

Reply:
@brentguf It's not visible to users yet - don't worry about missing translations for this one :)